src/main.ts

```diff
@@ -16,31 +18,33 @@ export interface TestArgs extends Argv {
 const command: Command = {
 	description: 'test your application',
 	register(options: OptionsHelper) {
+		options('a', {
+			alias: 'all',
+			describe: 'Indicates that all tests (both unit and functional) should be fun. By default, only unit tests are run.',
```

Comment:
I'm sure that all the tests will be fun, but I don't think we need a flag to indicate that.